playwright/e2e/crypto/backups.spec.ts

```diff
@@ -46,6 +50,59 @@ masTest.describe("Encryption state after registration", () => {
     });
 });
 
+masTest.describe("Key backup reset from elsewhere", () => {
+    masTest.skip(isDendrite, "does not yet support MAS");
+
+    masTest(
+        "Key backup is disabled when reset from elsewhere",
+        async ({ page, mailhog, request, masPrepare, homeserver }) => {
+            const testUsername = "alice";
+            const testPassword = "Pa$sW0rD!";
+
+            // there's a delay before keys are uploaded so the error doesn't appear immediately: use a fake
+            // clock so we can skip the delay
+            await page.clock.install();
+
+            await page.goto("/#/login");
+            await page.getByRole("button", { name: "Continue" }).click();
+            await registerAccountMas(page, mailhog.api, testUsername, "alice@email.com", testPassword);
+
+            await page.getByRole("button", { name: "Add room" }).click();
+            await page.getByRole("menuitem", { name: "New room" }).click();
+            await page.getByRole("textbox", { name: "Name" }).fill("test room");
+            await page.getByRole("button", { name: "Create room" }).click();
+
+            // @ts-ignore - this runs in the browser scope where mxMatrixClientPeg is a thing. Here, it is not.
+            const accessToken = await page.evaluate(() => mxMatrixClientPeg.get().getAccessToken());
+
+            const csAPI = new TestClientServerAPI(request, homeserver, accessToken);
```

Comment:
Why not use the *MatrixClient* to delete or to get the current backup info ?

Reply:
I was attempting to make it as much like a different client deleting the backup as possible, although what it ended up as isn't really much different from using the app's own  matrix client. It still felt a little bit cleaner to use the API separately rather than inject code into the app, but I can change it if that's preferred.